addons/html_editor/static/src/main/table/table_plugin.js

```diff
@@ -120,7 +122,10 @@ export class TablePlugin extends Plugin {
         targeted_nodes_processors: this.adjustTargetedNodes.bind(this),
         move_node_whitelist_selectors: "table",
         collapsed_selection_toolbar_predicate: (selectionData) =>
-            !!closestElement(selectionData.editableSelection.anchorNode, "td.o_selected_td"),
+            !!closestElement(
+                selectionData.editableSelection.anchorNode,
+                "td.o_selected_td, th.o_selected_td"
+            ),
```

Comment:
`o_selected_td` is only applied to td or th as of now why not:
```suggestion
            !!closestElement(selectionData.editableSelection.anchorNode, ".o_selected_td"),
```